src/ert/_c_wrappers/enkf/enkf_fs.py

```diff
@@ -218,6 +220,20 @@ def load_summary_data(
             return np.array([]), dates, loaded
         return np.concatenate(result, axis=1), dates, loaded
 
+    def load_summary_data_as_df(
+        self, summary_keys: List[str], realizations: List[int]
+    ) -> DataFrame:
+        data, x_axis, realizations = self.load_summary_data(summary_keys, realizations)
+        time_axis = x_axis
+        multi_index = MultiIndex.from_product(
+            [summary_keys, time_axis], names=["data_key", "axis"]
+        )
+        return DataFrame(
+            data=data.reshape(len(time_axis) * len(summary_keys), len(realizations)),
```

Comment:
Why is this reshape necessary?

Reply:
output of enkf_fs.load_summary_data is a 3D numpy array, but pandas expect a 2D array

Reply:
You were correct, the 3D array was old behaviour. Reshape removed :+1: